crates/agglayer-types/src/lib.rs

```diff
@@ -36,6 +37,12 @@ pub type Metadata = Digest;
 pub use pessimistic_proof::bridge_exit::NetworkId;
 use sp1_sdk::SP1VerificationError;
 
+pub mod primitives {
+    pub use agglayer_primitives::{
+        address, ruint, Address, Signature, SignatureError, B256, U256, U512,
+    };
+}
```

Comment:
Why not re-export the primitives wholesale?
```suggestion
pub use agglayer_primitives as primitives;
```

Reply:
Sure, why not?